### PR TITLE
Prd 5303 android sample app drop in checkout

### PR DESCRIPTION
**Pull request by inai-chinmay-flexiple**
Drop In Checkout Flow implemented

### CODE REVIEW

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import kotlinx.android.synthetic.main.fragment_present_checkout.*
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+
+class PresentCheckoutFragment : Fragment(R.layout.fragment_present_checkout), InaiCheckoutDelegate {
+
+    private val inaiBackendOrdersUrl: String = BuildConfig.InaiBaseUrl + "/orders"
```
Comment (inai-venthan) [inline app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:23]: Can we rename this to `baseUrl` without `Inai` prefix? Also can we make this change across all places?

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    private val authenticationString = NetworkRequestHandler.getEncodedAuthString(Config.inaiToken, Config.inaiPassword)
+    private var orderId = ""
+    private val orderMetadata: Map<String, JsonPrimitive> = mutableMapOf(
+        "test_order_id" to JsonPrimitive("test_order"),
+        "vat" to JsonPrimitive("6"),
+        "tax_percentage" to JsonPrimitive("12"),
+        "taxable_amount" to JsonPrimitive("50")
```
Comment (inai-venthan) [inline app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:30]: Lets remove this - across all places too if used elsewhere. Feel adding tax info to metadata could be misleading without proper context.

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    private fun getDataForOrders(): OrderPostData {
+        return OrderPostData(
+            amount = Config.amount,
+            currency = Config.currency,
+            customer = OrderCustomer(
+                email = "testdev@inai.io",
+                first_name = "Dev",
+                last_name = "Smith",
+                contact_number = "01010101010",
```
Comment (inai-venthan) [inline app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:125]: Can we make this consistent across too? Just for uniformity sake,
```suggestion
                email = "customer@example.com",
                first_name = "John",
                last_name = "Doe",
                contact_number = "01010101010",
```

**Review by inai-venthan — APPROVED** [review]
Minor comments, rest LGTM.

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt (hunk 1 above)
Comment (inai-chinmay-flexiple) [reply app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:23]: Done

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt (hunk 2 above)
Comment (inai-chinmay-flexiple) [reply app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:30]: Done

File: app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt (hunk 3 above)
Comment (inai-chinmay-flexiple) [reply app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt:125]: Done

**Review by inai-venthan — APPROVED** [review]

**Pull request merged**